src/components/dialogs/parameters/short-circuit-parameters.js

```diff
@@ -63,17 +64,12 @@ function getValue(param, key) {
 }
 
 const DropDown = ({ value, label, values, callback }) => {
-    const classes = useStyles();
     return (
         <>
-            <Grid item xs={8}>
-                <Typography component="span" variant="body1">
-                    <Box fontWeight="fontWeightBold" m={1}>
-                        <FormattedMessage id={label} />
-                    </Box>
-                </Typography>
```

Comment:
Why did you remove the `<Typography/>` tag here? You didn't remove it in others files.

Reply:
We now use a bold font from the style instead.